### PR TITLE
Accessibility within the search experience 

### PR DESCRIPTION
**Pull request by jclawson-coveo**
Blog post around accessibility

### CODE REVIEW

File: _posts/2020-08-25-Coveo-and-WCAG.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+![Example of Alt text]({{ site.baseurl }}/images/2020-08-25 Coveo-and-WCAG/Alt-text.png)
+
+As with ARIA Landmarks and alt text, another area that is important to address is color contrast. As front end developers are attempting to brand their websites, they need to take into account that not all color combinations are adhering to accessibility guidelines. A useful tool for checking compliance with contrasts can be found [on this website](https://snook.ca/technical/colour_contrast/colour.html#fg=01A982,bg=FFFFFF).
+
+## Conclusion
+
+The bottom line is that, yes, WCAG is important. It is important to understand the reason for the WCAG, because these users are depending on them to have a quality experience.
+
+Right now it’s normal for developers to check and double check their code for security vulnerabilities. Why shouldn’t we check and double check for accessibility features as well?
```
Comment (ariouxcoveo) [inline _posts/2020-08-25-Coveo-and-WCAG.md:72]: Double-check is a hyphenated compound word

File: _posts/2020-08-25-Coveo-and-WCAG.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+## Conclusion
+
+The bottom line is that, yes, WCAG is important. It is important to understand the reason for the WCAG, because these users are depending on them to have a quality experience.
+
+Right now it’s normal for developers to check and double check their code for security vulnerabilities. Why shouldn’t we check and double check for accessibility features as well? 
+
+Underneath all of the guidelines and rules, we are trying to provide an equal experience for everyone to find information. Coveo can make a major impact by circumventing the tedious navigation experience and instead provide the documentation with a search. In this day and age, inclusion should be one of our top priorities, and without an understanding of accessibility we are providing a lesser customer experience.
```
Comment (ariouxcoveo) [inline _posts/2020-08-25-Coveo-and-WCAG.md:74]: In this day and age, inclusion should be one of our top priorities, and without an understanding of accessibility**,** we are providing a lesser customer experience. : Add coma after accessibility and before we

File: _posts/2020-08-25-Coveo-and-WCAG.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+title: "Accessibility within the search experience "
+
+tags: [accessibility, search, compliance]
+
+author:
+  name: Jamie Lawson
+  bio: Solution Architect
+  image: Jamie.png
+---
+Working for a web based company, you’ve most likely heard the question: “Is your product WCAG compliant.” Having previously worked in sales, the answer for me was always something along the lines of “Yeah, sure, we are WCAG 2.0 compliant.” But, do you know what that actually means? Do you know what WCAG stands for? *hint Web Content Accessibility Guidelines
```
Comment (ariouxcoveo) [inline _posts/2020-08-25-Coveo-and-WCAG.md:13]: web-based: Missing hyphen

File: _posts/2020-08-25-Coveo-and-WCAG.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+-font colors and sizes
+-focus order
+-correct page titles
+-language automation 
+-page parsing
+
+## ARIA Landmarks, Alt Text, and Color
+
+ARIA landmarks are attributes you can add to elements in your page to define areas like the main content or a navigation region. Coveo has implemented ARIA Landmarks to help accessibility tools navigate through its components.These ARIA Landmarks are designed to identify and describe elements of a page.
+Below is a breakdown of the ARIA Landmarks found on the JavaScript Search Framework.
```
Comment (amoreauCoveo) [inline _posts/2020-08-25-Coveo-and-WCAG.md:46]: Add an extra line between 45 and 46 for a paragraph break

**Review by amoreauCoveo — COMMENTED** [review]
Nice stuff!

**Review by malaporte — APPROVED** [review]
Nice article, thanks!

**Review by amoreauCoveo — APPROVED** [review]

**Pull request merged**